### ISSUE TITLE
Authentication does not protect the new UI

### ISSUE THREAD

**Issue by suavelizard** [slanatech/swagger-stats#93]
When basic auth is configured: 
```
authentication: true,
      onAuthenticate: function (req, username, password) {
        return false
      },
```
`/swagger-stats/metrics` and `/swagger-stats/ui` will ask for authentication. However `/swagger-stats/ux` will not.

**Comment by ivashog** [comment]
When you publish #97 changes to npm?

**Comment by sv2** [comment]
Just published v.0.95.17 with #97 - thanks a lot !

**Comment by smashah** [comment]
Not sure if anyone else is facing this but if I use authentication and go directly to `/ux` it returns with `Authentication required`. I have to go to `/ui`, authenticate, then go to `/ux`.

Not a big deal, but worth noting.

**Comment by sv2** [comment]
Addressed v0.99.1
